nin/frontend/app/scripts/directives/editor/GraphEditorInputOutput.js

```diff
@@ -77,9 +77,9 @@ class GraphEditorInputOutput extends React.Component {
         strokeWidth: 2,
         onClick: event => {this.props.editor.startOrCompleteConnection(this.props.item);},
       }),
-      this.props.scale > 1.5 ? e('text', {
+      this.props.scale >= 1.5 ? e('text', {
         x: 0,
-        y: 20,
+        y: 10 * this.props.scale,
```

Comment:
Right now no one uses the lower zoom levels though.
If the entire system is up for a rewrite after revision anyways, I'd rather improve it for (what i think is) the common use case: the default zoom level.